tools/backups2datalad.req.txt

```diff
@@ -0,0 +1,11 @@
+# Python ~= 3.8
+boto3
+click >= 8.0.1
+click-loglevel ~= 0.2
+#dandi >= 0.21.0
+dandi @ git+https://github.com/dandi/dandi-cli@api-02
```

Comment:
note: I guess we better finalize/release that PR before merging/deploying this PR